docs/api/getting-started.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: /
+sidebar_position: 1
+---
+
+# Getting Started with the API
+
+Welcome to the Saiki API. This guide will walk you through the essential first steps to begin interacting with your Saiki agent programmatically.
```

Comment:
i feel like this part needs some rewording

we should explain a bit about the API

something like this maybe:
```
Saiki agents also have in-built APIs you can use to communicate with your agent.

This guide will walk you through the steps needed to set up the APIs for your Saiki agent.
```